pkg/proxy/proxy.go

```diff
@@ -111,7 +111,35 @@ func newRedirect(port uint16, source ProxySource, id string) *Redirect {
 	}
 }
 
-func (r *Redirect) updateAccounting(t accesslog.FlowType, verdict accesslog.FlowVerdict) {
+func (r *Redirect) DeriveEndpointInfo(ip net.IP, info *accesslog.EndpointInfo) bool {
```

Comment:
Remove. This doesn't seem to be called anywhere.

Reply:
Did I miss to commit something?

```
pkg/proxy/logger/logger.go-             // if IP belongs to the cluster.
pkg/proxy/logger/logger.go-             if node.GetIPv4ClusterRange().Contains(ip) {
pkg/proxy/logger/logger.go-                     // If endpoint cannot be found, set to cluster identity
pkg/proxy/logger/logger.go:                     if !producer.DeriveEndpointInfo(ip, info) {
pkg/proxy/logger/logger.go-                             fillIdentity(info, identity.ReservedIdentityCluster)
pkg/proxy/logger/logger.go-                     }
pkg/proxy/logger/logger.go-             } else {
--
pkg/proxy/logger/logger.go-             }
pkg/proxy/logger/logger.go-
pkg/proxy/logger/logger.go-             if node.GetIPv6ClusterRange().Contains(ip) {
pkg/proxy/logger/logger.go:                     if !producer.DeriveEndpointInfo(ip, info) {
pkg/proxy/logger/logger.go-                             fillIdentity(info, identity.ReservedIdentityCluster)
pkg/proxy/logger/logger.go-                     }
pkg/proxy/logger/logger.go-             } else {
```